lib/senders/index.js

```diff
@@ -160,7 +151,7 @@ module.exports = function (log, config, error, bounces, translator, sender) {
       },
       sendPasswordChangedNotification: function (emails, account, opts) {
         var primaryEmail = account.email
-        var ccEmails = getSecondaryEmails(emails)
+        var ccEmails = getVerifiedSecondaryEmails(emails)
```

Comment:
Looks like we have a running list of items in https://github.com/mozilla/fxa-auth-server/issues/1725, appending it.